### ISSUE TITLE
Why create benchmarcks instead of use existing benchmarcks ? 

### ISSUE THREAD

**Issue by LifeIsStrange** [Oktosha/servo-image-benchmark#2]
All is in the title and sorry If I say Something of stupid but I think it's a lack of time.

**Comment by Oktosha** [comment]
I haven't found any existing benchmarks (I mean [benchmark tests](https://doc.rust-lang.org/book/benchmark-tests.html) when I say benchmarks here and in todo) for the Rust image cargo. I think that they don't exist because in the dev IRC channel it was suggested to create a pull request to [image cargo](https://github.com/PistonDevelopers/image) with my performance tests.

**Comment by Oktosha** [comment]
I haven't found a benchmarking utility for libpng, it will be great if you will provide me with a link. I need an utility, not just test results, because I'd like to run all the performance tests on the same machine.

**Comment by Oktosha** [comment]
You are right that there is a benchmarking utility tjbench to test libjpeg-turbo [performance](http://www.libjpeg-turbo.org/About/Performance) and I should mention it in the todo. But I still need to write at least a shell-script to run it with proper params on proper images. Also I haven't tried tjbench yet and can't guarantee anything about it. You see, it's the last point in todo, so it isn't precise yet.

But you are right that this point in the todo should be more precise.

**Comment by LifeIsStrange** [comment]
I'm infinitely sorry for the late !! 
I don't really know this benchmarcks but they exists 
See this libjpeg turbo which speak about benchmarcks http://www.libjpeg-turbo.org/About/Performance
Maybe this : http://personals.ac.upc.edu/alvarez/media/media_benchmarks.html
Also a post about the importance of using the gpu for image decoding : http://semiaccurate.com/2014/04/17/amd-makes-jpegs-suck-less/
